dev-aws/kafka-shared/iam_cerbos-audit.tf

```diff
@@ -2,32 +2,56 @@ resource "kafka_topic" "iam_cerbos_audit_v1" {
   name               = "auth.iam-cerbos-audit-v1"
   replication_factor = 3
   partitions         = 10
-  config = {
+  config             = {
```

Comment:
:thinking: thought for future: maybe we could add `pre-commit` to this repo to run `terraform fmt` for us to keep things tidy

Reply:
Yea, that would be good

Reply:
> 🤔 thought for future: maybe we could add `pre-commit` to this repo to run `terraform fmt` for us to keep things tidy

https://github.com/utilitywarehouse/kafka-cluster-config/pull/137